<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1310</width>
    <height>882</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="close_pg">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>닫기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hide_pg">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>최소화</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="collect_gr">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>631</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel, QLineEdit, QComboBox, QProgressBar {border: 1px solid black;
			background-color: white;
			qproperty-alignment: AlignCenter;}

QPushButton {border: 1px solid black;
			background-color: #cdcdcd;
			qproperty-alignment: AlignCenter;}


QGroupBox{
border: 1px solid black;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTableWidget" name="keyword_table2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>221</width>
       <height>411</height>
      </rect>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>index</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>키워드</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>시작날짜</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="start_date">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="set_date">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>설정완료</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="end_date">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>끝날짜</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>381</width>
       <height>411</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="to_word">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>워드로 출력</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="news_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>361</width>
        <height>361</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>언론사</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>제목</string>
       </property>
      </column>
     </widget>
     <widget class="QComboBox" name="filter">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>전체</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>경쟁사</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>자사</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>업계</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>필터</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="log2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>460</y>
       <width>511</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>프로그램 메세지</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reset_date">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>재설정</string>
     </property>
    </widget>
    <zorder>reset_date</zorder>
    <zorder>keyword_table2</zorder>
    <zorder>label_2</zorder>
    <zorder>start_date</zorder>
    <zorder>set_date</zorder>
    <zorder>end_date</zorder>
    <zorder>label_3</zorder>
    <zorder>groupBox</zorder>
    <zorder>log2</zorder>
    <zorder>label_5</zorder>
   </widget>
   <widget class="QPushButton" name="config_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>설정</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="config_gr">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>590</y>
      <width>631</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel, QLineEdit, QComboBox, QProgressBar {border: 1px solid black;
			background-color: white;
			qproperty-alignment: AlignCenter;}

QPushButton {border: 1px solid black;
			background-color: #cdcdcd;
			qproperty-alignment: AlignCenter;}


QGroupBox{
border: 1px solid black;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTableWidget" name="keyword_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>221</width>
       <height>411</height>
      </rect>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>index</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>키워드</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="set_detail_btn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>400</y>
       <width>381</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>설정완료</string>
     </property>
    </widget>
    <widget class="QLabel" name="curr_keyword">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="set_detail_close">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>430</y>
       <width>381</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>다음</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_39">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>유형</string>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="category">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>유형1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>유형2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>키워드 추가</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="keyword_edit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>441</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="add_keyword">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>추가</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_41">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>댓글 수집</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comment">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>130</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>사용안함</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>사용</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>프로그램 메세지</string>
     </property>
    </widget>
    <widget class="QLabel" name="log">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>460</y>
       <width>511</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="set_date_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>140</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>설정완료</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1310</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
